extension/Popup/popup.js

```diff
@@ -26,19 +39,28 @@ chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
           <div class="six wide column"><p>Ingredient Name</p></div>
           <div class="four wide column"><p>Quantity</p></div>
           <div class="four wide column"><p>Unit</p></div>
-        </div>`
+        </div>`;
 
-    response.ingredients.forEach(ingObj => {
-      let ingHTML = `<div class="row">
+    response.ingredients.forEach((ingObj) => {
+      const ingHTML = `<div class="row">
           <div class="six wide column"><p>${ingObj.name}</p></div>
           <div class="four wide column"><p>${ingObj.quantity}</p></div>
           <div class="four wide column"><p>${ingObj.unit}</p></div>
           <div class="two wide column"><i class="remove icon"></i></div>
-        </div>`
+        </div>`;
       htmlString += ingHTML;
-    })
-    htmlString += `</div>`;
-    $('body').append(htmlString)
-  })
-})
+    });
+
+    // const buttonHtml = '<button type"button">Send Recipe to Sous</button>';
+    const buttonHtml = '<button type="button" id="btnSendRecipe" name="btnSendRecipe">Send Recipe to Sous</button></div>';
+    htmlString += buttonHtml;
+
+    htmlString += '</div>';
+
+    $('body').append(htmlString);
 
+    $('#btnSendRecipe').click(() => {
```

Comment:
add the msg: 'create grocery list' to the response obj

then do chrome.extension.sendMessage(response)